### ISSUE TITLE
Could not file or assembly xunit.execution.AspNet on CTP6

### ISSUE THREAD

**Issue by ardalis** [xunit/aspnet.xunit#15]
Trying to run xunit tests on a new ASPNET 5 class library project using VS2015 CTP6.  My project.json:
{
    "version": "1.0.0-*",
    "dependencies": {
        "xunit": "2.0.0-*",
        "xunit.runner.aspnet": "2.0.0-*"
    },
    "commands": {
        "test": "xunit.runner.aspnet"
    },
    "frameworks" : {
        "aspnet50" : { 
            "dependencies": {
            }
        },
        "aspnetcore50" : { 
            "dependencies": {
                "System.Runtime": "4.0.20-beta-22523"
            }
        }
    }
}

This pulled in xunit 2.0.0-rc3-build2902 and xunit.runner.aspnet 2.0.0-rc3-build52.

When I go to the appropriate folder and run k test I get:
![image](https://cloud.githubusercontent.com/assets/782127/6491312/6270abcc-c27a-11e4-968e-d9392dd9940b.png)
Discovering: Core.Tests
System.IO.FileNotFoundException: Could not load file or assembly 'xunit.execution.AspNet' or one of its dependencies.

UPDATE - RESOLVED:
Specified these builds and the tests work:
"dependencies": {
        "xunit": "2.0.0-rc3-build2880",
        "xunit.runner.aspnet": "2.0.0-rc3-build51"
    }

**Comment by bradwilson** [comment]
Make sure you are NOT referencing our CI build NuGet server. Post RC3 we've removed ASP.NET 5 support, and you're pulling one of these later builds.

(We will immediately be re-introducing a 2.1 beta with ASP.NET 5 support; we just can't ship it in the RTM, since it's not RTM itself.)

**Comment by radenkozec** [comment]
@ardalis Had a same issue. Now I can run from CMD with k test. 

Have you got it work from Visual Studio Test Explorer using "xunit.runner.visualstudio": "2.0.0.0-rc3-build1046" ?
I have not got luck with that.

**Comment by giggio** [comment]
Also having the same problem with 2.1 beta:

````json
    "dependencies": {
        "xunit": "2.0.0-*",
        "xunit.runner.aspnet": "2.1.0-*"
    }
````

It restores fine, the only problem is when running.

**Comment by bradwilson** [comment]
You should be using 2.1.0 for both xunit and the runner, per the instructions:
http://xunit.github.io/docs/getting-started-aspnet.html

**Comment by giggio** [comment]
It works now. Was this just modified? I tried pulling xunit 2.1 before but it was not there yet.

**Comment by bradwilson** [comment]
I pushed it all out to NuGet on Saturday night (Pacific time), and then tweeted about it. https://twitter.com/xunit/status/579459289309282304

**Comment by radenkozec** [comment]
It is fixed with  2.1 beta. I now can run test from Visual Studio. Thanks
